definitions/features/mongo.rb

```diff
@@ -0,0 +1,46 @@
+class Features::Mongo < ForemanMaintain::Feature
+  MONGO_DB_CONFIG = '/etc/mongod.conf'.freeze
+  # FIXME: remote mongo won't have this config
+  # decide based on if Pulp is using it
+
+  metadata do
+    label :mongo
+
+    confine do
+      file_exists?(MONGO_DB_CONFIG)
+    end
+  end
+
+  def data_dir
+    '/var/lib/mongodb'
+  end
+
+  def config_files
+    [
+      MONGO_DB_CONFIG
+    ]
+  end
+
+  def dump(target)
+    execute!("mongodump --host localhost --out #{target}")
```

Comment:
will this be supported for external mongo?

Reply:
yes, I'm working on it